src/app/consumption-estimation/consumption-estimation.component.ts

```diff
@@ -0,0 +1,206 @@
+import { Component, OnInit } from '@angular/core';
+import { FormControl,FormGroup,Validators,FormBuilder, FormArray } from '@angular/forms'
+
+
+@Component({
+  selector: 'app-consumption-estimation',
+  templateUrl: './consumption-estimation.component.html',
+  styleUrls: ['./consumption-estimation.component.css']
+})
+export class ConsumptionEstimationComponent implements OnInit {
+
+  serviceChoiceForm!:FormGroup;
+  familyAdressCompoundForm!: FormGroup;
+  consumptionDetailsForm!: FormGroup;
+  meterTypeForm!:FormGroup;
+  service_step = false;
+  familyAdress_step = false;
+  meterType_step=false;
+  consumption_step = false;
```

Comment:
Are these used if u use the step var anyway? Might want to change it to an enum if u intent to really use it like this.